lib/ts/recipe/passwordless/utils.ts

```diff
@@ -13,52 +13,138 @@
  * under the License.
  */
 
+import { CountryCode } from "libphonenumber-js";
+import { parsePhoneNumber } from "libphonenumber-js/min";
 import { FeatureBaseConfig, NormalisedBaseConfig } from "../../types";
 import { normaliseAuthRecipe } from "../authRecipe/utils";
-import { defaultEmailValidator } from "../emailpassword/validators";
-import { Config, NormalisedConfig } from "./types";
+import { Config, NormalisedConfig, SignInUpFeatureConfigInput } from "./types";
 import { RecipeInterface } from "./types";
-import { defaultPhoneNumberValidator } from "./validators";
+import {
+    defaultPhoneNumberValidator,
+    defaultPhoneNumberValidatorForCombinedInput,
+    defaultEmailValidator,
+    defaultEmailValidatorForCombinedInput,
+} from "./validators";
 
 export function normalisePasswordlessConfig(config: Config): NormalisedConfig {
+    if (!["EMAIL", "PHONE", "EMAIL_OR_PHONE"].includes(config.contactMethod)) {
+        throw new Error("Please pass one of 'PHONE' or 'EMAIL' as the contactMethod");
```

Comment:
Change message to `"Please pass one of 'PHONE', 'EMAIL' or 'EMAIL_OR_PHONE' as the contactMethod"`